packages/plugins/type-designer/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@oscd-plugins/type-designer",
 	"private": true,
-	"version": "0.0.0",
+	"version": "0.1.0",
```

Comment:
Do we already version our product? We need to come up with a team agreement on this

Reply:
We iterate version to trigger the github action and deploy a version of the plugin. This will be the first iteration, and I changed the version number to 1.0.0, because it is our first functional major version.